buddyboss/buddyboss-platform
# 850 - Fixed the JS error for memberpress stripe credit card field

### All Submissions:

* x ] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?





### Changes proposed in this Pull Request:



Fixes #850 .

### How to test the changes in this Pull Request:

1. Activate Memberpress
2. Create New Membership and add payment gateway strip.
3. Open that form and see credit card detail fields
4. See credit card field works properly

### Proof Screenshots or Video
https://prnt.sc/sb5dzo


### Other information:




### Changelog entry

> Fixed the JS error for memberpress stripe credit card field
